app/services/jwt/token_descriptor.rb

```diff
@@ -0,0 +1,18 @@
+module Jwt::TokenDecryptor
```

Comment:
JWT is a abbr
Jwt::TokenDecryptor class_name, jwt/token_descriptor.rb name of file. ?!